plugins/html-elements-plugin/src/main/java/ru/sbtqa/tag/pagefactory/reflection/HtmlReflection.java

```diff
@@ -306,7 +234,7 @@ public static WebElement findElementInBlockByTitle(Page page, String blockPath,
      * @return first found block object
      * @throws java.util.NoSuchElementException if couldn't find any block
      */
-    public static HtmlElement findBlock(Page page, String blockPath) throws NoSuchElementException {
+    public HtmlElement findBlock(Page page, String blockPath) throws NoSuchElementException {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove the declaration of thrown exception 'org.openqa.selenium.NoSuchElementException' which is a runtime exception. ([squid:RedundantThrowsDeclarationCheck](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3ARedundantThrowsDeclarationCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=101&issues=AWbDgFYRLpqqSUlctfyC&open=AWbDgFYRLpqqSUlctfyC)

[ISSUE_KEY]: AWbDgFYRLpqqSUlctfyC